xml/System/DateOnly.xml

```diff
@@ -1615,13 +1647,14 @@
         <Parameter Name="result" Type="System.DateOnly" RefType="out" />
```

Comment:
@gewarren @joelmartinez the `<Parameter Name="s" ...>` element was not added to this API. Bug?

Reply:
I see it here: https://github.com/dotnet/dotnet-api-docs/blob/dd358f5f27a75e2a7bcc084e084103858d6b27d4/xml/System/DateOnly.xml#L1629

Reply:
Okay thanks. * Sigh *... GitHub playing games again.